src/index.js

```diff
@@ -0,0 +1,102 @@
+// State Definer
+
+const state = {
+    temperature: 50
+};
+
+// Helper functions
+
+const temperatureLogic = (tempUpBackground, gardenBox) => {
+    if (state.temperature <= 49) {
+        tempUpBackground.className = 'forties';
+        gardenBox.textContent = '☃️☃️☃️☃️☃️☃️☃️☃️☃️☃️☃️☃️☃️';
+    } else if (50 <= state.temperature && state.temperature <= 59) {
+        tempUpBackground.className = 'fifties';
+        gardenBox.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    } else if (60 <= state.temperature && state.temperature <= 69) {
+        tempUpBackground.className = 'sixties';
+        gardenBox.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    } else if (70 <= state.temperature && state.temperature <= 79) {
+        tempUpBackground.className = 'seventies';
+        gardenBox.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else {
+        tempUpBackground.className = 'eighties';
+        gardenBox.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    };
+};
+
+const weatherLogic = (weather, sky) => {
+    if (weather.value === 'sunny') {
+        sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+    } else if (weather.value === 'cloudy') {
+        sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+    } else if (weather.value === 'rainy') {
+        sky.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+    } else if (weather.value === 'snowy') {
+        sky.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+    }
+};
+
+//Event Functions 
+
+const increaseTemperature = () => {
+    const tempUp = document.getElementById('temperature')
+    state.temperature += 1;
+    tempUp.textContent = `${state.temperature}℉`;
+
+    const tempUpBackground = document.querySelector('#temperature')
+    const gardenBox = document.getElementById('garden-box')
+
+    temperatureLogic(tempUpBackground, gardenBox)
+};
+
+const decreaseTemperature = () => {
+    const tempDown = document.getElementById('temperature')
+    state.temperature -= 1;
+    tempDown.textContent = `${state.temperature}℉`;
+
+    const tempUpBackground = document.querySelector('#temperature')
+    const gardenBox = document.getElementById('garden-box')
+
+    temperatureLogic(tempUpBackground, gardenBox)
+};
```

Comment:
This functionality works great! Is there a way you can think to refactor your code so there's less duplicated logic across these two functions?